service/src/main/java/bio/terra/cbas/common/MethodUtil.java

```diff
@@ -56,4 +56,8 @@ public static MethodSourceEnum convertToMethodSourceEnum(String methodSource)
 
     throw new UnknownMethodSourceException(methodSource);
   }
+
+  public static Boolean verifyIfPrivate(String methodUrl) {
+    return true;
```

Comment:
Just noting it here - as discussed in standup this should be set to false as default.